superset-frontend/cypress-base/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "@types/querystringify": "^2.0.0",
-    "cypress": "^10.11.0",
+    "cypress": "^11.2.0",
```

Comment:
This is a bump to cypress 11 and not 12, right?

Reply:
Yeah, I was running tests on 11 and then was going to to 12, but I think I'll just stick to 11 in this PR.

Reply:
Ok this looks good then